### PR TITLE
All bar one comment functions converted to drizzle

### PR DESCRIPTION
**Pull request by JohnAllenTech**
# ✨ Codu Pull Request 💻

Codu Logo

## Pull Request details

- More drizzle migrations
- Pretty straightforward I see some areas to improve but have kept with the current implementation to avoid potentially breaking the sql

## Any Breaking changes

- None

## Associated Screenshots

- None

## Summary by CodeRabbit

- **New Features**
	- Enhanced comment functionality including creation, editing, and deletion.
	- Improved notification system for comments and replies.
	- Updated like mechanism to efficiently handle likes and dislikes.
- **Refactor**
	- Optimized database queries for posts, comments, and notifications for better performance.

### CODE REVIEW

File: server/api/router/comment.ts [hunk 1]
```diff
@@ -11,6 +11,8 @@ import {
   NEW_COMMENT_ON_YOUR_POST,
   NEW_REPLY_TO_YOUR_COMMENT,
 } from "../../../utils/notifications";
+import { comment, notification, like } from "@/server/db/schema";
+import { and, eq } from "drizzle-orm";
 
 export const commentRouter = createTRPCRouter({
   create: protectedProcedure
```
[the remark this answers is not in the document]
Comment (JohnAllenTech) [reply server/api/router/comment.ts]: I wish bot.. I tried that and it didnt work

**Comment by NiallJoeMaher** [comment]
Some conflicts in here now 🥳

File: server/api/router/comment.ts [hunk 2]
```diff
@@ -33,6 +36,7 @@ export const commentRouter = createTRPCRouter({
           postId,
           body,
           parentId,
+          updatedAt: now,
```
Comment (JohnAllenTech) [inline server/api/router/comment.ts:39]: Always passing this now instead of depending on DB

**Comment by JohnAllenTech** [comment]
> Some conflicts in here now 🥳

Should be good to go now. Fixed the updatedAt thing also

**Review by NiallJoeMaher — APPROVED** [review]
🌮

**Pull request merged**